src/main/java/edu/cornell/kfs/concur/ConcurConstants.java

```diff
@@ -48,5 +48,11 @@
     public static final String ACCEPT_SUB_FOLDER_NAME = "accept";
     public static final String REJECT_SUB_FOLDER_NAME = "reject";
     public static final String FORWARD_SLASH = "/";
-    public static final String CONCUR_PROCESSED_FILE_DATE_FORMAT = "yyyy-MM-dd-HHmmss";
+    public static final String CONCUR_DATE_FORMAT = "yyyy-MM-dd";
+    public static final String CONCUR_PROCESSED_FILE_DATE_FORMAT = CONCUR_DATE_FORMAT + "-dd-HHmmss";
```

Comment:
I was looking at this PR to help with my analysis for some other Concur-related user stories, and I noticed that this constant is now being set to "yyyy-MM-dd-dd-HHmmss" instead of "yyyy-MM-dd-HHmmss". Is this intentional?